kernel/src/processbuffer.rs

```diff
@@ -26,12 +26,18 @@
 
 use core::cell::Cell;
 use core::marker::PhantomData;
+use core::mem::size_of;
 use core::ops::{Deref, Index, Range, RangeFrom, RangeTo};
 
 use crate::capabilities;
 use crate::process::{self, ProcessId};
 use crate::ErrorCode;
 
+// the offset where the buffer data starts
+// - skip the flags field (1 byte)
+// - skip the len field (size_of::<u32>() bytes)
+const BUFFER_OFFSET: usize = 1 + size_of::<u32>();
```

Comment:
Can we move this into the impl ala https://github.com/tock/tock/blob/master/kernel/src/process_standard.rs#L1300?